android-components/components/feature/prompts/src/main/res/values/strings.xml

```diff
@@ -20,25 +20,37 @@
     <!-- Text for password field in an authentication dialog. -->
     <string name="mozac_feature_prompt_password_hint">Password</string>
     <!-- Negative confirmation that we should not save the new or updated login -->
-    <string name="mozac_feature_prompt_dont_save">Don’t save</string>
+    <string name="mozac_feature_prompt_dont_save" moz:removedIn="123" tools:ignore="UnusedResources">Don’t save</string>
+    <!-- Negative confirmation that we should not save the new or updated login -->
+    <string name="mozac_feature_prompt_dont_save_1">Not now</string>
```

Comment:
Sorry to ask this of you, but could you use `_2` for all of these?

Reply:
It will be a lot of changes. Is there a specific reason for this?

Reply:
In general, I think you want to be incrementing the strings. `_1` is generally implied with the first string. This is more about avoiding confusion to know which string came beforehand.